### ISSUE TITLE
Not clear how to use new BrowserFetcher class

### ISSUE THREAD

**Issue by drmrbrewer** [puppeteer/puppeteer#9139]
### Bug description

The release notes for [version 19.0.0](https://github.com/puppeteer/puppeteer/releases/tag/v19.0.0) notes the following breaking change: "deprecate `createBrowserFetcher` in favor of `BrowserFetcher`".

But the [docs for the BrowserFetcher class](https://pptr.dev/api/puppeteer.browserfetcher) don't really explain how to use it.

Till now I've only ever imported `puppeteer` at the top of my code:

    import puppeteer from 'puppeteer';

and from that I've created a `browserFetcher` via the following call:

    const browserFetcher = puppeteer.createBrowserFetcher();

But I'm left wondering what I need to import in order to access `BrowserFetcher` so that I can create a `browserFetcher` instance with the new API:

    const browserFetcher = new BrowserFetcher();

Probably just being dumb but it would be good if the docs could explain this for us dumbos.  Thanks!

### Puppeteer version

19.0.0

### Node.js version

16.17.0

### npm version

8.15.0

### What operating system are you seeing the problem on?

Linux

**Comment by jrandolf** [comment]
Hey @drmrbrewer, thanks for the issue.

For version 19.0.0, we planned to deprecate `createBrowserFetcher` since all it did was just call the constructor to `BrowserFetcher`, however we are planning to undeprecate it **only for `puppeteer`**. 

Tl;dr, `puppeteer` uses a lot of defaults under the hood to make the coding experience more friendly, so we plan to implement defaults into the `createBrowserFetcher` method.

So continue using `createBrowserFetcher` if you are using `puppeteer` :)

**Comment by drmrbrewer** [comment]
@jrandolf OK thanks, I'll carry on using `puppeteer.createBrowserFetcher()` :-)

Out of interest, what is the use-case for the standalone `BrowserFetcher` class and how would you go about using it in a simple node script?

**Comment by pSnehanshu** [comment]
At the moment, all documentation for `createBrowserFetcher` and `BrowserFetcher` stands removed. I haven't been following the development of puppeteer, so don't know what happened. Can anyone point me in the right direction?
